src/Avalonia.FreeDesktop/DBusMenuExporter.cs

```diff
@@ -255,19 +255,19 @@ private int GetId(NativeMenuItemBase item)
                         if (mod.KeyModifiers.HasAllFlags(KeyModifiers.Meta))
                             lst.Add("Super");
                         lst.Add(item.Gesture.Key.ToString());
-                        return VariantValue.ArrayOfVariant((VariantValue[]) [VariantValue.Array(lst)]);
+                        return Variant.FromArray(new Array<Array<string>>([lst]));
```

Comment:
Here the type is changing between _array of array of string_ and _array of variant_.

Reply:
To keep the _array of array of string_, the change should be:
```cs
var lst = new Array<string>();
// ...
return Variant.FromArray(new Array<Array<string>>([lst]));
```
to:
```cs
var lst = new Array<string>();
// ...
return new Array<Array<string>>([lst]);
```